fastlane/lib/fastlane/actions/zip.rb

```diff
@@ -74,7 +81,8 @@ def self.example_code
           'zip(
             path: "MyApp.app",
             output_path: "Latest.app.zip",
-            verbose: false
+            verbose: false,
+            symlinks: false
```

Comment:
It doesn't make much sense to add the param with its default value to an existing example. Better create a new, third one, that shows it changing the default behavior.